### PR TITLE
Feature/api updates fe

### PR DESCRIPTION
**Pull request by Jyckle**
@seandough3rty if you could review this that would be great. Added the functionality on the front end side for deleting things. all of the desired API routes are in the dataservice.ts file

### CODE REVIEW

File: Angular/src/app/language-add/language-add.component.html [hunk 1]
```diff
@@ -1,3 +1,7 @@
+<button (click)="goBack()" class="button"><i class="material-icons">arrow_back</i></button>
+
 <h1> Add A Language </h1>
 <input class ="input-text" [(ngModel)]="languageName" placeholder="Language Name">
-<a routerLink=".." (click)="addLanguage()"> <i class="material-icons">add_circle_outline</i> </a>
\ No newline at end of file
+<a routerLink=".." (click)="addLanguage()"> \
```
Comment (SeanDougherty) [inline Angular/src/app/language-add/language-add.component.html]: Is this backslash supposed to be here?

File: Angular/src/app/paradigm-edit/paradigm-edit.component.html [hunk 2]
```diff
@@ -8,8 +11,13 @@ <h1>
...
 		<h2> Root Words </h2> 
 		<ul>
-		  <li *ngFor="let root of roots$; let index = index" (click)="onSelect(index)">
-		    <a>{{ root }}</a>
+		  <li *ngFor="let root of roots$; let index = index" class="root-item-container">
+		    <div (click)="onSelect(index)" class="root-item">
+		    	<a>{{ root }}</a> 
+		    </div> 
+		    <div class="root-delete">
+		    	<a (click)="deleteWord(index)"><i class="material-icons">delete</i></a>
```
Comment (SeanDougherty) [inline Angular/src/app/paradigm-edit/paradigm-edit.component.html]: This is nitpicky, and I'll leave the implementation up to you, but should the (click)="function()" references be in the same tag types? Either both inside of the < a > tag or both inside of the < div > tag?

**Review by SeanDougherty — CHANGES_REQUESTED** [review]
Looks good overall, Just a few details that I think should be double checked. Also, that's an alarming number of lines being deleted. Are we aware of exactly is being removed? Just want to be sure before we proceed. Thanks John!

File: Angular/src/app/language-add/language-add.component.html (hunk 1 above)
Comment (Jyckle) [reply Angular/src/app/language-add/language-add.component.html]: No it is not, good catch. Fixed it in the most recent commit

File: Angular/src/app/paradigm-edit/paradigm-edit.component.html (hunk 2 above)
Comment (Jyckle) [reply Angular/src/app/paradigm-edit/paradigm-edit.component.html]: Good call here as well, I think it's better to stay consistent with this. I updated it in the most recent commit

**Comment by Jyckle** [comment]
I think I fixed the issues you pointed out in the most recent commit. Let me know but hopefully that works! Additionally, the deleted files are from removing the "Dashboard" folder which is no longer needed. So although this is a large deletion, it is supposed to happen!

**Pull request merged**